examples/python/custom_tools/hosted_server.py

```
@@ -0,0 +1,197 @@
+
+This module provides a base class for creating hosted MCP servers
+that can handle requests and integrate with the MCP protocol.
+
+Note: This is an example implementation. To use it, ensure you have installed:
+  pip install praisonaiagents[mcp] starlette>=0.27.0
+"""
+
+import asyncio
+import logging
+from typing import Dict, Any, Optional, List, Callable
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)